packages/xo-web/src/xo-app/dashboard/health/index.js

```diff
@@ -517,7 +518,15 @@ const ALARM_ACTIONS = [
 
 const HANDLED_VDI_TYPES = new Set(['system', 'user', 'ephemeral'])
 
-@connectStore(() => {
+const THIRTY_DAYS = Date.now() - 30 * 24 * 60 * 60 * 1000
```

Comment:
Don't use `Date.now()` here, otherwise `THIRTY_DAYS` will be evaluated at runtime and your `Date.now()` will be a static time.
```suggestion
const THIRTY_DAYS = 30 * 24 * 60 * 60 * 1000
```